internal/runner/common/unit_resolver.go

```diff
@@ -747,7 +757,7 @@ func (r *UnitResolver) flagDependencyIncludes(dependency *Unit, unitPath string,
 }
 
 // flagExcludedUnits iterates over a unit slice and flags all units that are excluded based on the exclude block.
-func (r *UnitResolver) flagExcludedUnits(l log.Logger, opts *options.TerragruntOptions, units Units) Units {
+func (r *UnitResolver) flagExcludedUnits(l log.Logger, opts *options.TerragruntOptions, reportInstance *report.Report, units Units) Units {
```

Comment:
```suggestion
func (r *UnitResolver) flagExcludedUnits(l log.Logger, opts *options.TerragruntOptions, r *report.Report, units Units) Units {
```

This is odd to me. Why aren't we passing `r` down to avoid conflicting with the package name. Is there a special reason we're calling it `reportInstance` here?